src/ric/21250.c

```diff
@@ -1570,6 +1576,11 @@ void RicUpdatePlayerEntities(void) {
     }
 }
 
+#if defined(VERSION_US)
```

Comment:
My suggestion here would be to #ifdef the individual entries. What if someone wants to mod and change E_WHIP to true and E_CRASH_HYDROSTORM to false

Reply:
I think with so many lines, it's better to have a single ifdef.

If someone wants to mod the whip and hydrostorm to be different, they can type "true" and "false" in the two B_MAKE lines, and replace the MAYBE call.

Separately, there should be a different word used for MAYBE, which describes the field it controls. But for the question about how to ifdef, I think one ifdef at the top is better than doing them all over.

Reply:
Sure that works